doc/installation.rst

```diff
@@ -164,7 +160,7 @@ The requirements to run Spyder are:
 
 * `Pickleshare <https://github.com/pickleshare/pickleshare>`_ -- To show import completions in the Editor and Consoles.
 
-* `PyZMQ <https://github.com/zeromq/pyzmq>`_ -- To run introspection services in the Editor asynchronously.
+* `PyZMQ <https://github.com/zeromq/pyzmq>`_ -- CLient for the Language Server Protocol.
```

Comment:
I didn't notice this, but there's a little error here: `CLient` -> `Client`

Reply:
Thanks; I'll update it in #87